packages/server/graphql/mutations/removeAllSlackAuths.ts

```diff
@@ -0,0 +1,42 @@
+import {GraphQLNonNull} from 'graphql'
+import getRethink from '../../database/rethinkDriver'
+import {requireSU} from '../../utils/authorization'
+import RemoveAllSlackAuthsPayload from '../types/RemoveAllSlackAuthsPayload'
+import {GQLContext} from '../graphql'
+
+const removeAllSlackAuths = {
+  type: GraphQLNonNull(RemoveAllSlackAuthsPayload),
+  description: 'Remove Slack integrations for all users',
+  resolve: async (_source, _args, {authToken}: GQLContext) => {
+    const r = await getRethink()
+    const now = new Date()
+    //AUTH
+    requireSU(authToken)
+
+    // RESOLUTION
+    const allSlackAuths = await r
+      .table('SlackAuth')
+      .filter({isActive: true})
+      .run()
+
+    const allSlackAuthIds = allSlackAuths.map(({id}) => id)
+    const [slackAuthRes, slackNotificationRes] = await Promise.all([
+      r
+        .table('SlackAuth')
+        .getAll(r.args(allSlackAuthIds))
+        .update({botAccessToken: null, isActive: false, updatedAt: now})
+        .run(),
+      r
+        .table('SlackNotification')
+        .delete()
+        .run()
+    ])
+    const data = {
+      slackAuthRes: JSON.stringify(slackAuthRes),
```

Comment:
As this function will only be used internally, I thought it'd be best to show the stringified response so we can see how many Slack Auths / notifications have been updated / removed

Reply:
good thinking! since this will be private, can you move this to the intranetSchema?